README.md

```diff
@@ -35,13 +35,19 @@ pip install -e .
 
 ### A2C
 * [Cart pole](alf/examples/ac_cart_pole.gin).
+
 * [Atari games](alf/examples/ac_breakout.gin). Need to install python package atari-py for atari game environments.
+
 * [Simple navigation with visual input](alf/examples/ac_simple_navigation.gin). Follow the instruction at [SocialRobot](https://github.com/HorizonRobotics/SocialRobot) to install the environment.
 
 ### PPO
 * [PR2 grasping state only](alf/examples/ppo_pr2.gin). Follow the instruction at [SocialRobot](https://github.com/HorizonRobotics/SocialRobot) to install the environment.
-* [Humonoid](alf/examples/ppo_bullet_humanoid.gin). Need to install python pybullet>=2.5.0 for the environment.
+* [Humonoid](alf/examples/ppo_bullet_humanoid.gin). Learning to walk using pybullet humanoid environment. Need to install python pybullet>=2.5.0 for the environment.
```

Comment:
this version is higher than the version specified in tf-agent submodule, copied over here: https://github.com/HorizonRobotics/alf/pull/104/commits/63c0268a38ca006a2fc2e1437ad36b622d4b5166

Should I update the copied over version number to this one?

Reply:
I am not sure pybullet 2.4.2 will work. I trained the model using pybullet 2.5.0. To be safe, we should use 2.5.0, unless you are willing to spend time testing 2.4.2.

Reply:
pybullet 2.4.2 won't work as I tested. 2.5.0 is a safe version